packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -33,37 +40,90 @@ withVersions('express', 'express', version => {
       })
     })
 
-    beforeEach(async () => {
-      requestBody = sinon.stub()
-      appsec.enable(new Config({ appsec: { enabled: true, rules: path.join(__dirname, 'express-rules.json') } }))
-    })
-
-    afterEach(() => {
-      appsec.disable()
-    })
-
     after(() => {
       server.close()
       return agent.close({ ritmReset: false })
     })
 
-    it('should not block the request without an attack', async () => {
-      const res = await axios.get(`http://localhost:${port}/?key=value`)
+    describe('Blocking', () => {
+      beforeEach(async () => {
+        requestBody = sinon.stub()
+        appsec.enable(new Config({ appsec: { enabled: true, rules: path.join(__dirname, 'express-rules.json') } }))
+      })
+
+      afterEach(() => {
+        appsec.disable()
+      })
+
+      it('should not block the request without an attack', async () => {
+        const res = await axios.get(`http://localhost:${port}/?key=value`)
 
-      expect(requestBody).to.be.calledOnce
-      expect(res.data).to.be.equal('DONE')
+        expect(requestBody).to.be.calledOnce
+        expect(res.data).to.be.equal('DONE')
+      })
+
+      it('should block the request when attack is detected', async () => {
+        try {
+          await axios.get(`http://localhost:${port}/?key=testattack`)
+
+          return Promise.reject(new Error('Request should not return 200'))
+        } catch (e) {
+          expect(e.response.status).to.be.equals(403)
+          expect(e.response.data).to.be.deep.equal(JSON.parse(json))
+          expect(requestBody).not.to.be.called
+        }
+      })
     })
 
-    it('should block the request when attack is detected', async () => {
-      try {
-        await axios.get(`http://localhost:${port}/?key=testattack`)
+    describe('Api Security', () => {
+      let config
 
-        return Promise.reject(new Error('Request should not return 200'))
-      } catch (e) {
-        expect(e.response.status).to.be.equals(403)
-        expect(e.response.data).to.be.deep.equal(JSON.parse(json))
-        expect(requestBody).not.to.be.called
-      }
+      beforeEach(() => {
+        config = new Config({
+          appsec: {
+            enabled: true,
+            rules: path.join(__dirname, 'api_security_rules.json'),
+            apiSecurity: {
+              enabled: true,
+              requestSampling: 1.0
+            }
+          }
+        })
+      })
+
+      afterEach(() => {
+        appsec.disable()
+      })
+
+      it('should get the schema', async () => {
+        appsec.enable(config)
+
+        const res = await axios.post(`http://localhost:${port}/`, { key: 'value' })
+
+        await agent.use((traces) => {
+          const span = traces[0][0]
+          expect(span.meta).to.haveOwnProperty('_dd.appsec.s.req.body')
+          expect(span.meta['_dd.appsec.s.req.body']).to.be.equal('H4sIAAAAAAAAA4uuVspOrVSyiraIrY0FAJDZQ4oNAAAA')
```

Comment:
This test is working because it is executed only in Ubuntu, but should do the same changes that you made here: https://github.com/DataDog/dd-trace-js/pull/3685/files#diff-3050e8b7ea9f6b36aad60624e737503306c4fb65efd5e6429ed03786918ca5c0R313